infiniband-diags/ibroute.c

```diff
@@ -321,7 +321,7 @@ static int dump_lid(char *str, int strlen, int lid, int valid)
 static const char *dump_unicast_tables(ib_portid_t *portid, int startlid,
 				       int endlid)
 {
-	char lft[IB_SMP_DATA_SIZE] = { 0 };
+	unsigned char lft[IB_SMP_DATA_SIZE] = { 0 };
```

Comment:
Shouldn't this should be uint8_t because it is data from the network?

Reply:
Dobe, was changed to uint8_t .

Reply:
Is it ok now?